### ISSUE TITLE
RxSwift/RxCocoa 4.5 compatibility

### ISSUE THREAD

**Issue by seanrucker** [RxSwiftCommunity/RxGesture#88]
The dependencies in the podspec prevent it from being compatible with the latest versions of RxSwift and RxCoccoa:
```ruby
s.dependency 'RxSwift', '~> 4.4.0'
s.dependency 'RxCocoa', '~> 4.4.0'
```

Changing them to this would solve the problem:
```ruby
s.dependency 'RxSwift', '~> 4.4'
s.dependency 'RxCocoa', '~> 4.4'
```

**Comment by seanrucker** [comment]
RxSwift/RxCocoa are required for Xcode 10.2 and Swift 5 compatibility so it may make sense to address this in #87.

**Comment by inchtime** [comment]
I can't build my project now.
Are there any other solutions?

**Comment by EstebanVallejo** [comment]
Temporary, until this is fixed, you can change the framework swift version with the following post install script.  
 
```
post_install do |installer|
        
        installer.pods_project.targets.each do |target|
            if target.name == 'RxGesture'
                target.build_configurations.each do |config|
                    config.build_settings['SWIFT_VERSION'] = '4.0'
                end
            end
        end
       
end
```

**Comment by adm73** [comment]
My settings as follow and works.

```
    pod 'RxSwift',    '~> 4.0'
    pod 'RxCocoa',    '~> 4.0'
    pod 'RxGesture',  '~> 2.0'
```

**Comment by seanrucker** [comment]
@adm73 check your Podfile.lock that is likely resolving to using RxSwift/RxCocoa 4.4.

**Comment by pclemente** [comment]
Hi!

While the authors update the library, I have forked it and added Compatibility with RxSwift/RxCocoa 4.5. You can use it from here: https://github.com/pclemente/RxGesture

Hope it helps :)

**Comment by hampusiggstrom** [comment]
Now fixed in 2.2.0 release
